kuma/static/js/wiki-samples.js

```diff
@@ -94,7 +94,7 @@
                     // create icon
                     var $icon = $('<i />', { 'class': 'icon-' + sampleCodeHost, 'aria-hidden': 'true' });
                     // create text
-                    var $text = 'Open in ' + this + ' ';
+                    var $text = gettext('Open in ') + this + ' ';
```

Comment:
This should instead use [ngettext](https://docs.djangoproject.com/en/1.9/topics/i18n/translation/#interpolate), like ``var $text = ngettext('Open in %(site)s ', this);``, so that languages where the words come in a different order will be supported.